src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -3078,6 +3081,178 @@ GUI_TEST_CLASS_DEFINITION(test_7659) {
+    // Return to the "Query Designer" window.
+    // Select the subunit of the Primer element on the Scene.
+    //     Its parameters appear in the Property Editor.
+    // Change the Parameters:
+    //     Product size ranges             100-300
+    //     Number to return                3
+    //     Max repeat mispriming           10
+    //     Max template mispriming         10
+    //     Max 3' stability                10
+    //     Pair max repeat mispriming      20
+    //     Pair max template mispriming    20
+    // Run Schema.
+    //     The "Run Schema" dialog appears.
+    // Click Run.
+    // Expected: the following annotations:
+    //     Result 1    199..218    complement(297..316)
+    //     Result 2    40..59      complement(297..316)
+    //     Result 3    39..58      complement(297..316)
+    GTFileDialog::openFile(os, testDir + "_common_data/primer3/only_primer.uql");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    runSchema(os, testDir + "_common_data/primer3/all_settingsfiles.fa");
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 1  (0, 2)", {{850, 869}, {1022, 1041}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 2  (0, 2)", {{22, 41}, {199, 218}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 3  (0, 2)", {{850, 869}, {993, 1012}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 4  (0, 2)", {{452, 471}, {610, 629}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 5  (0, 2)", {{610, 629}, {786, 805}});
+    GTUtilsProject::closeProject(os, true);
+    GTUtilsMdi::activateWindow(os, "Query Designer - NewSchema");
+
+    clickOn(GTUtilsQueryDesigner::getItemCenter(os, "Primer"));
+    auto table = GTWidget::findTableView(os, "table");
+    {  // Product size ranges.
+        clickOn(GTTableView::getCellPosition(os, table, 1, 2));
+        GTLineEdit::setText(os, GTWidget::findWidgetByType<QLineEdit*>(os, table, "7667"), "100-300");
+    }
+    {  // Number to return.
+        clickOn(GTTableView::getCellPosition(os, table, 1, 3));
+        GTSpinBox::setValue(os, GTWidget::findWidgetByType<QSpinBox*>(os, table, "7667-0"), 3);
+    }
+    setDouble(os, table, 4, 10);  // Max repeat mispriming.
+    setDouble(os, table, 5, 10);  // Max template mispriming.
+    setDouble(os, table, 6, 10);  // Max 3' stability.
+    setDouble(os, table, 7, 20);  // Pair max repeat mispriming.
+    setDouble(os, table, 8, 20);  // Pair max template mispriming.
+    runSchema(os);
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 1  (0, 2)", {{199, 218}, {297, 316}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 2  (0, 2)", {{40, 59}, {297, 316}});
+    GTUtilsAnnotationsTreeView::checkAnnotationRegions(os, "Result 3  (0, 2)", {{39, 58}, {297, 316}});
+}
+
+GUI_TEST_CLASS_DEFINITION(test_7667_0) {
```

Comment:
Naming is non-symmetrical: I would suggest using `test_7667_1` and `test_7667_2`

Reply:
✓